sdk/tables/azure-data-tables/pom.xml

```diff
@@ -38,6 +38,11 @@ Licensed under the MIT License.
       <artifactId>azure-core</artifactId>
       <version>1.17.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-identity</artifactId>
+      <version>1.3.1</version> <!-- {x-version-update;com.azure:azure-identity;dependency} -->
```

Comment:
This needs to be a `<scope>test</scope>` dependency. `azure-identity` isn't allowed as a runtime dependency.